core/src/main/kotlin/com/strumenta/kolasu/parsing/ParseTree.kt

```diff
@@ -117,6 +116,44 @@ fun Token.getOriginalText(): String {
     return this.inputStream.getText(interval)
 }
 
-fun Node.getText(): String? = parseTreeNode?.getOriginalText()
+fun Node.getText(): String? = if (origin != null) {
```

Comment:
I think you could do something like `origin?.apply/let { }`

Reply:
Didn't know that, sure